tests/integration/components/mascots/mascot-filters-test.js

```diff
@@ -7,12 +7,6 @@ module('Integration | Component | mascots/mascot-filters', function (hooks) {
   setupRenderingTest(hooks);
 
   test('We show 9 filters', async function (assert) {
-    /*
-      TODO:
-
-      After we upgrade ember-source to v3.24+, we can remove
-      the line `this.owner.setupRouter();`.
-    */
```

Comment:
Here, calling `this.owner.setupRouter();` is necessary because we don't inject the `router` service in the component.